### PR TITLE
add command line option

### PR DESCRIPTION
**Pull request by JensWehner**
add command line option for gpus to xtp.

The implementation is not perfect, basically I use sort of a global variable, but in some sense OPENMP does the same, so maybe it is okay at least for now.

### CODE REVIEW

**Comment by JensWehner** [comment]
@votca-bot format

**Comment by JensWehner** [comment]
@votca-bot changelog: add command line option for number of gpus

File: src/libxtp/openmp_cuda.cc [hunk 1]
```diff
@@ -465,7 +487,8 @@ Eigen::MatrixXd OpenMP_CUDA::getReductionVar() {
   for (Index i = 0; i < Index(gpus_.size()); i++) {
     GPU_data& gpu = gpus_[i];
     gpu.activateGPU();
-    cpus_[i].reduce() = *(gpu.temp.back());
+    Eigen::MatrixXd temp=*(gpu.temp.back());
```
Comment (felipeZ) [inline src/libxtp/openmp_cuda.cc:490]: Out of curiosity why do you need this temp array here?

**Review by felipeZ — APPROVED** [review]

File: src/libxtp/openmp_cuda.cc (hunk 1 above)
Comment (JensWehner) [reply src/libxtp/openmp_cuda.cc]: Good point I do not :)

File: src/libxtp/openmp_cuda.cc [hunk 2]
```diff
@@ -465,7 +487,8 @@ Eigen::MatrixXd OpenMP_CUDA::getReductionVar() {
   for (Index i = 0; i < Index(gpus_.size()); i++) {
     GPU_data& gpu = gpus_[i];
     gpu.activateGPU();
-    cpus_[i].reduce() = *(gpu.temp.back());
+    Eigen::MatrixXd temp = *(gpu.temp.back());
+    cpus_[i].reduce() += temp;
```
Comment (JensWehner) [inline src/libxtp/openmp_cuda.cc:491]:
```suggestion
    cpus_[i].reduce() = *(gpu.temp.back());
```

**Pull request merged**